apps/backend/src/server.ts

```diff
@@ -3,3 +3,12 @@ const port = process.env.PORT ?? 5678;
 app.listen(port, () => {
   console.log(`listening at http://localhost:${port}`);
 });
+
+function shutdown(signal: string) {
+    console.log(`Received event: ${signal}`)
+    app.close(()=>{
+      process.exit()
+    })
+}
+process.on('SIGINT',shutdown)   // CTRL-C
+process.on('SIGTERM', shutdown) // docker kill
```

Comment:
Thank you! It works without --init now, but the format and lint don't seem to be applied. Could you explicitly add :void as the return type for the function to avoid ESLint errors, and format the code as well?

Reply:
I will certainly fix the formatting: my bad. I can fix the return type as well. I generally don't use typescript so I missed that as well